spec/support/shared_examples/authenticator_base.rb

```diff
@@ -8,7 +8,7 @@
     end
 
     it "considers an authentically-signed request to be inauthentic when it's too old or too far in the future" do
-      [-301, 301].each do |time_offset|
+      [-301, 601].each do |time_offset|
```

Comment:
Why was this changed?  5 minutes is still the `ALLOWED_DRIFT_SECONDS` value.

It would probably be clearer if this were:
```
drift = MAuth::Client::AuthenticatorBase::ALLOWED_DRIFT_SECONDS
[-(drift + 1),  drift + 1].each do |time_offset|
```

Reply:
Honestly, I'm not sure why this needed to be bumped.